insta/src/lib.rs

```diff
@@ -62,15 +62,7 @@
 //!
 //! Note that `cargo-insta` is entirely optional.  You can also just use insta
 //! directly from `cargo test` and control it via the `INSTA_UPDATE` environment
-//! variable.  The default is `auto` which will write all new snapshots into
-//! `.snap.new` files if no CI is detected so that `cargo-insta` can pick them
-//! up.  The following other modes are possible:
-//!
-//! - `auto`: the default. `no` for CI environments or `new` otherwise
-//! - `always`: overwrites old snapshot files with new ones unasked
-//! - `unseen`: behaves like `always` for new snapshots and `new` for others
-//! - `new`: write new snapshots into `.snap.new` files
-//! - `no`: does not update snapshot files at all (just runs tests)
```

Comment:
This is the older text, and a few lines later there's a slightly different version. So this now links to that verison